### ISSUE TITLE
ModuleNotFoundError: No module named 'bcolz.carray_ext'

### ISSUE THREAD

**Issue by forkbabu** [stefan-jansen/bcolz-zipline#36]
Python 3.10.12

git commit id: [8a00889](https://github.com/stefan-jansen/bcolz-zipline/commit/8a008894871a08681cffb4f08845090d6d6f9a00)

![image](https://github.com/stefan-jansen/bcolz-zipline/assets/8037297/b84b5070-0706-4a2a-9238-2cfdf38da772)

![image](https://github.com/stefan-jansen/bcolz-zipline/assets/8037297/0593c324-909e-47be-a20e-0013298b0194)

@stefan-jansen

**Comment by gnzsnz** [comment]
I'm facing a similar issues while building wheels for linux/arm64(it works for linux/amd64 tough)


```
Error compiling Cython file:
2612.7       ------------------------------------------------------------
2612.7       ...
2612.7               # Create the final container and fill it
2612.7               out = carray([], dtype=newdtype, cparams=self.cparams,
2612.7                              expectedlen=newlen,
2612.7                              rootdir=rootdir, mode='w')
2612.7               if newlen < ilen:
2612.7                   rsize = isize / newlen
2612.7                                 ^
2612.7       ------------------------------------------------------------
2612.7
2612.7       bcolz/carray_ext.pyx:1685:26: Cannot assign type 'double' to 'npy_intp'
2612.7       Compiling bcolz/carray_ext.pyx because it changed.
2612.7       [1/1] Cythonizing bcolz/carray_ext.pyx
```

`Cython.Compiler.Errors.CompileError: bcolz/carray_ext.pyx`

**Comment by amoskaliov** [comment]
I had the same problem as @gnzsnz with the original bcolz package. Since it's an error of Cython I installed an old version of the package (pip install Cython==0.29.26) and managed to install bcolz then.

**Comment by gnzsnz** [comment]
@amoskaliov  thanks for your input.

unfortunatelly in my case setting  Cython==0.29.26 did not work.

**Comment by stefan-jansen** [comment]
Should be closed by #41
